setup.env

```diff
@@ -1,13 +1,12 @@
 TASKBIN="${PWD}/bin"
 GOPATH="${PWD}/cache/go"
 GOBIN="${GOPATH}/bin"
-TASK="${TASKBIN}/task"
 
 # install task
-[[ -x "${TASK}" ]] || sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -b "${TASKBIN}"
+[[ -x "${TASKBIN}/task" ]] || sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -b "${TASKBIN}"
 
 # export custom go environment
-export GOPATH="${PWD}/cache/go"
+export GOPATH
```

Comment:
Does this actually work? Or does need to look like `export GOPATH="${GOPATH}"`?

Reply:
Line 2 does already assign a value to GOPATH, and it is optional but not required for exporting:

```
export: export [-fn] [name[=value] ...] or export -p
    Set export attribute for shell variables.
    
    Marks each NAME for automatic export to the environment of subsequently
    executed commands.  If VALUE is supplied, assign VALUE before exporting.
 [...]
```